packages/angular-eslint/src/index.ts

```diff
@@ -72,15 +76,17 @@ both CJS and ESM in very natural ways.
 */
 export default {
   configs,
+  tsParser: parser,
```

Comment:
Why did you add this? I don't want to export a parser from another package, that should be imported directly

Reply:
Only for convenience and consistency.

---

Because there were no **base** configs exported I had to rewrite them in my code.

Importing your package gave me some insights about what your package was using, thanks to autocompletion:
```ts
import angularPlugin from 'angular-eslint';

angularPlugin.***
```
For instance, I easily found that there were a `templatePlugin` and a `tsPlugin`.
And also a `templateParser`, but no `tsParser`...
I didn't find any info in the doc about the parser that was used, so I had to dig in the code to find out I was supposed to use the parser from `typescript-eslint`.
So I thought it would be convenient and consistent to also export a `tsParser` property.